### ISSUE TITLE
v8.10Installation Wizard rejects correct mysql dbusr & dbpwd

### ISSUE THREAD

**Issue by schokobass** [owncloud/core#18063]

**Comment by LukasReschke** [comment]
Thanks a lot for reporting issues back to us!

We need some more information to properly support you with your issue!

Can I ask you to follow our guidelines for submitting bugs as described here: https://github.com/owncloud/core/blob/master/CONTRIBUTING.md

**Comment by schokobass** [comment]
Hi, I would like to install OC 8.1.0 in a Host Europe webspace (WebPack 4L with  MySQL 5.5.38-1~dotdeb.0-log and PHP5.5).

I have tried both upload options (by side with setup-owncloud.php, and the .zip). In both cases I get stuck  at the same point when trying to create a MySQL-based installation:

After reaching the page for defining the OC administrator name & pwd, the DB type, and the DB credentials (dbhost, dbusr, dbname, dbpwd), in any case I just get the error message

"MySQL/MariaDB-Benutzername und/oder -Passwort sind nicht gültig Du musst entweder ein existierendes Benutzerkonto oder das Administratorenkonto angeben."

These things I already did test without success:

1.
set the MySQL dbpwd for the global dbusr again -> I am sure that I do type in the correct string;

2.
trying the setup step for a db object not existing and trying it with an existing db instance with own dbusr & dbpwd (refer to https://doc.owncloud.org/server/8.1/admin_manual/installation/installation_wizard.html#required-settings -> "Storage & Database" -> "The database you want to use must already be installed, and you must have the database administrative user and password for all databases except SQLite, which does not have an admin user. Enter any arbitrary name for the ownCloud Database name. This must be a database that does not already exist.");

3.
at the end, I even tried to install the v8.1.0 tar.bz2 under Ubuntu 14.04 locally confirming to https://doc.owncloud.org/server/7.0/admin_manual/installation/source_installation.html - and ended at exactly the same point.

4.
In the directory /data, no htaccesstest.txt file is to be found. Under Ubuntu 14.04, all file and directory rights are set accordingly to the description in the user manual. I do not get any error message besides the rejection of my MySQL credentials.

I would be very grateful for any suggestion, what could be wrong . . .
